### PR TITLE
Interim update for dev quickstart

### PR DESCRIPTION
**Pull request by ptone**
Just something to get a few more people started till docs get better

### CODE REVIEW

**Comment by takluyver** [comment]
Thanks, this looks sensible. I'll let @minrk have a look before we merge.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 The Jupyter HTML notebook is a web-based notebook environment for interactive computing.
 
+
+Dev quickstart:
+
+* Create a virtual env (ie jupyter-dev)
+* Clone this repo and cd into it
+* pip install -r requirements.txt
+* pip install -e .
+* ensure that you have node/npm install (ie brew install node)
```
Comment (Carreau) [inline README.md]: brew only on os x, but OK :-)

File: README.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+* npm install -g 'less@<3.0'
```
Comment (minrk) [inline README.md]: less doesn't need to be installed globally anymore. Nothing does, except npm.

File: requirements.txt [hunk 3]
```diff
@@ -6,3 +6,4 @@
 -e git+https://github.com/ipython/ipython.git#egg=ipython
 -e git+https://github.com/ipython/ipython_kernel.git#egg=ipython_kernel
 -e git+https://github.com/jupyter/jupyter_nbconvert.git#egg=jupyter_nbconvert
+invoke
```
Comment (minrk) [inline requirements.txt]: invoke isn't used anymore

File: README.md (hunk 1 above)
Comment (minrk) [inline README.md]: and ensure you have node/npm should come *before* pip install.

**Comment by drocco007** [comment]
@takluyver pointed out that for Debian/Ubuntu systems, if you're installing the system `node` you need to use the `nodejs-legacy` package and *not* the `node` package. It would be nice to mention this in the instructions as this is easy to have incorrect and almost impossible to discover form the failure messages.

**Comment by ptone** [comment]
So the invoke + less stuff was needed earlier today, this is all now already a bit simpler

File: README.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
 
 The Jupyter HTML notebook is a web-based notebook environment for interactive computing.
 
+Dev quickstart:
+
+* Create a virtual env (ie jupyter-dev)
+* ensure that you have node/npm install (ie brew install node on OS X)
```
Comment (minrk) [inline README.md]: s/install/installed

**Comment by takluyver** [comment]
@minrk this was updated, can you take another look?

**Comment by minrk** [comment]
:+1:

**Pull request merged**